content-services/content-sources-frontend
# Add Formatting/Vscode/Typescript support for Eslint

To test this: 
- Open this branch with VS-Code, you should see a prompt regarding extensions, install any that are required. 
- (optional) update your settings to format on save. 
- Go make a change to a file, intentionally make line widths and other unnecessary spacing.
- Hit save (or Format), ..., prettier should fix any issues found.

To run the lint/format from the command line, simple do: npm run lint:fix && npm run format
